commit_check/util.py

```
@@ -112,35 +131,80 @@ def cmd_output(commands: list) -> str:
     :returns: Get `str` output.
     """
     result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)